device-types/Palo Alto/PA-3420.yaml

```diff
@@ -0,0 +1,85 @@
+---
+manufacturer: Palo Alto
+model: PA-3420
+slug: palo-alto-pa-3420
+part_number: PA-3420
+u_height: 1
+is_full_depth: false
+weight: 7.00
+weight_unit: kg
+airflow: front-to-rear
+comments: '[Palo Alto 3400 Series Data Sheet](https://www.paloaltonetworks.com/apps/pan/public/downloadResource?pagePath=/content/pan/en_US/resources/datasheets/pa-3400-series-pan-os-10-2)'
+console-ports:
+  - name: console
+    type: rj-45
+power-ports:
+  - name: PS1
+    type: iec-60320-c14
+    maximum_draw: 190
+    allocated_draw: 155
+  - name: PS2
+    type: iec-60320-c14
+    maximum_draw: 190
+    allocated_draw: 155
```

Comment:
It appears that the PSU is hot swappable. Thus it should be modelled under `module-bays`, and then the associated PSU module should be modelled in the module-types folder if they do not already exist there

If I am wrong about this please let me know and mark this as resolved